clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientV3Provider.groovy

```diff
@@ -26,9 +26,11 @@ import org.openstack4j.openstack.OSFactory
 class OpenstackClientV3Provider extends OpenstackClientProvider {
 
   Token token
+  List<String> regions
 
-  OpenstackClientV3Provider(OSClient.OSClientV3 client) {
+  OpenstackClientV3Provider(OSClient.OSClientV3 client, List<String> regions) {
```

Comment:
I thought it would be nice to be able to limit your regions via configuration in v3.  If we don't want to do that, I can easily remove.  Just needed to start somewhere and leaned towards, more configuration.